src/profile.ts

```diff
@@ -41,14 +41,27 @@ async function addUserInfo (uok: UsernameOrKaid): Promise<void> {
 		"Average spinoffs received": averageSpinoffs,
 		"Total badges": totalBadges,
 		"Inspiration badges": totals.inspiration,
-		"More info": `<a href="${userEndpoint}/profile?${uok.type}=${uok.id}&format=pretty" target="_blank">API endpoint</a>`
+		"More info": `API endpoint`
 	} as { [key: string]: string | number; };
 
 	for (const entry in entries) {
-		table.innerHTML += `<tr>
-				<td class="user-statistics-label">${entry}</td>
-				<td>${entries[entry]}</td>
-			</tr>`;
+		const trTag = document.createElement("tr");
+		const tdLabelTag = document.createElement("td");
+		tdLabelTag.className = "user-statistics-label";
+		tdLabelTag.textContent = entry;
+		trTag.appendChild(tdLabelTag);
+		const tdEntryTag = document.createElement("td");
+		if (entry !== "More info") {
+			tdEntryTag.textContent = `${entries[entry]}`;
```

Comment:
This and line 60 could just be
```
tdEntryTag.textContent = entries[entry];
```

Reply:
If I was to remove them, this error will appear:
```
      TS2322: Type 'string | number' is not assignable to type 'string | null'.
  Type 'number' is not assignable to type 'string | null'.
```

I'll see if I can work out how to solve it though.